CITATION.cff

```diff
@@ -1,290 +1,431 @@
+  orcid: https://orcid.org/0000-0002-9667-8654
+  affiliation: Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen Centre for Astroparticle Physics
+  city: Erlangen, FR
+- family-names: Morcuende
+  given-names: Daniel
+  email: dmorcuen@ucm.es
+  orcid: https://orcid.org/0000-0001-9400-0922
+  affiliation: GAE-UCM
+  city: Madrid, ES
+- family-names: Nakashima
+  given-names: Kaori
+  email: kaori.nakashima@fau.de
+  affiliation: Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen Centre for Astroparticle Physics
+  city: Erlangen, FR
+- family-names: Nigro
+  given-names: Cosimo
+  email: cosimo.nigro@ifae.es
+  orcid: https://orcid.org/0000-0001-8375-1907
+  affiliation: IFAE
+  city: Barcelona, SP
+- family-names: Nöthe
+  given-names: Maximilian
+  email: maximilian.noethe@tu-dortmund.de
+  affiliation: TU Dortmund University
+  city: Dortmund, Germany
+- family-names: Olivera-Nieto
+  given-names: Laura
+  email: laura.olivera-nieto@mpi-hd.mpg.de
+  orcid: https://orcid.org/0000-0002-9105-0518
+  affiliation: Max Planck Institut für Kernphysik, MPIK
+  city: Heidelberg, DE
+- family-names: Owen
+  given-names: Ellis
+  email: ellis.owen@mpi-hd.mpg.de
+  affiliation: National Tsing Hua University Institute of Astronomy
+  city: Hsinchu, Taiwan
+  orcid: https://orcid.org/0000-0003-1052-6439
+- family-names: Panny
+  given-names: Sebastian
+  email: sebastian.panny@outlook.com
+  orcid: https://orcid.org/0000-0001-5770-3805
+  affiliation: Universität Innsbruck
+  city: Innsbruck, AT
+- family-names: Papadopoulos
+  given-names: Dimitri
+  email: dimitri.papadopoulos@cea.fr
+  orcid: https://orcid.org/0000-0002-1242-8990
+  affiliation: CEA
+  city: Saclay, FR
+- family-names: Pintore
+  given-names: Fabio
+  email: fabio.pintore@inaf.it
+  affiliation: INAF/IASF PALERMO
+  city: Palermo, Italy
+  orcid: https://orcid.org/0000-0002-3869-2925
+- family-names: Poon
+  given-names: Helen
+  email: china_108@yahoo.com
+  orcid: https://orcid.org/0000-0002-6004-4270
+  affiliation: Max Planck Institut für Kernphysik, MPIK
+  city: Heidelberg, DE
+- family-names: Remy
+  given-names: Quentin
+  email: quentin.remy@mpi-hd.mpg.de
   orcid: https://orcid.org/0000-0002-8815-6530
-- given-names: Léa
-  family-names: Jouvin
-- given-names: Atreyee
-  family-names: Sinha
-  affiliation: Universidad Complutense de Madrid EMFTEL, UCM, Plaza de la Ciencias,
-    28040 Madrid, Spain
+  affiliation: Max Planck Institut für Kernphysik, MPIK
+  city: Heidelberg, DE
+- family-names: Robitaille
+  given-names: Thomas
+  email: thomas.robitaille@gmail.com
+  orcid: https://orcid.org/0000-0002-8642-1329
+  affiliation: Max Planck Innsbruck for Astronomy
+  city: Heidelberg, DE
+- family-names: Ruiz
+  given-names: José Enrique
+  email: jer@iaa.es
+  orcid: https://orcid.org/0000-0003-3274-4445
+  affiliation: Instituto de Astrofísica de Andalucía - CSIC
+  city: Granada, Spain
+- family-names: Saha
+  given-names: Lab
+  email: labsaha@ucm.es
```

Comment:
@labsaha  is not at the UCM anymore.

Reply:
At the time of the contribution, it was... I will contact Lab to ask